src/main/java/duke/UI.java

```diff
@@ -0,0 +1,95 @@
+package duke;
+
+/**
+ * The UI class handles user interface interactions.
+ */
+public class UI {
```

Comment:
I believe the letter I should not be capitalized since UI is an abbreviation for User Interface.

Reply:
Thank you! Will amend in the next version.